<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entrywindow</class>
 <widget class="QDialog" name="Entrywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUPID</string>
  </property>
  <widget class="QCheckBox" name="studentCheckBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Student</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="adminCheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Administrator</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_carleton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Loginbutton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
